packages/core/package.json

```diff
@@ -27,9 +27,6 @@
     "upload-integration-test": "bin/upload-lambda.js local.bundle.zip",
     "deploy-integration-test":
       "yarn build-integration-test && yarn upload-integration-test",
-    "build-boilerplate": "bin/build-boilerplate.sh",
-    "upload-boilerplate": "bin/upload-boilerplate.sh",
-    "postpublish": "bin/build-boilerplate.sh && bin/upload-boilerplate.sh",
```

Comment:
The `postpublish` part has been moved to the bottom of `REPO_ROOT/scripts/publish.sh`. The boilerplate needs updating not only when there's a new version of `core` but also when `legacy-scripting-runner` has a new version.